plugins/embark-slither/babel.config.js

```diff
@@ -0,0 +1,48 @@
+/* global module */
+
+module.exports = function (api) {
+  const node = {
+    ignore: [
+      'src/lib/modules/pipeline/babel-loader-overrides.js',
+      'src/lib/modules/pipeline/webpack.config.js'
+    ],
+    plugins: [
+      'babel-plugin-macros',
```

Comment:
I believe a lot of this can be removed when https://github.com/embark-framework/embark/pull/1299/files#diff-fc10a6682269c1fefc2abe05d1500b8aR14 lands. @michaelsbradleyjr may need to chime in on the specifics of what can be removed or should be changed.

Reply:
Yes totally agree, I suggested same changes in https://github.com/embark-framework/embark/pull/1294